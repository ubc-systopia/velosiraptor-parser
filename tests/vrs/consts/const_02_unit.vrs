const FOOBAR: int = 1234;

segment Abc {
  const FOOBAR: int = 5432;
  const FOOBAR: int = 5432;
}