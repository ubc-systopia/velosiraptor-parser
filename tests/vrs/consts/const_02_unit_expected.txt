VelosiParseTree(tests/vrs/consts/const_02_unit.vrs)
---------------------------------------------
const FOOBAR : int = 1234;

segment Abc {
  const FOOBAR : int = 5432;
  const FOOBAR : int = 5432;
}


---------------------------------------------
